slippage_accounting.md

```diff
@@ -0,0 +1,51 @@
+# Solver Slippage Accounting
+
+Slippage accounting is performed on a per settlement/transaction basis according to the following two primary
+components. The SQL source code can be found
+on [GitHub](https://github.com/cowprotocol/solver-rewards/blob/main/queries/dune_v2/period_slippage.sql)
+or [Dune Analytics](https://dune.com/queries/2421375)
+
+## 1. Batch-wise Token Imbalance
+
+The token balance sheet represents a classified ledger all incoming and outgoing token transfers relative to the
+settlement contract. Classification categories are `USER_{IN/OUT}`, `AMM_{IN/OUT}` and `PHANTOM_TRANSFER`
+
+### Transfer Type Classification
+
+In all cases `IN` represents settlement contract as `recipient` and `OUT` as `sender`!
+
+- `USER_{IN/OUT}` transfers are those emitted by the Settlement contract's Trade Event (with `USER_IN` adjusted for
+  fees).
+- `AMM_{IN/OUT}` classification is assigned to all on-chain transfers that are NOT user transfers
+- `PHANTOM_TRANSFER` represents token transfers associated with _internalized buffer trades_. In brief, these are
+  transfers related to AMM interactions that were "skipped" for the purpose of gas optimization in favour of using the
+  settlement contract's internal holdings. The process for including this "off-chain" data in rather involved so we
```

Comment:
typo:

`...data in rather...`   -> is

Also next line

`dedicate and entire` -> an

Reply:
```suggestion
  settlement contract's internal holdings. The process for including this "off-chain" data is rather involved so we
```